css/css-backgrounds/reference/border-image-image-type-003-ref.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Reference Test</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+
+  <style>
+  div
+    {
+      background-image: linear-gradient(to top, blue, orange);
+      height: 140px;
+      margin: 10px;
+      width: 140px;
+    }
+
+  img
+    {
+      left: 30px;
+      position: relative;
+      top: 30px;
+    }
+  </style>
+
+  <p>Test passes if there is no red visible on the page.
+
+  <div><img src="../support/1x1-white.png" width="80" height="80" alt="Image download support must be enabled"></div>
```

Comment:
Does this need to be an image? Why not just an element with a white background?

Reply:
No.
Okay. 
New commit coming up in 5 min.